Menu/Menu.js

```diff
@@ -1,10 +1,18 @@
-
 const toggleMenu = () => {
   // Toggle the "menu--open" class on your menu refence. 
+  menu.classList.toggle('menu--open');
+  /* added animation to menu */
+ if(menu.classList.contains("menu--open"))
+  {
+    TweenMax.to(menu, 1, {left: 0})
```

Comment:
There are quite a few problems here.

1. When you are closing the menu. menu has a `display:none`. You can't do animation on thing that isn't displaying

2. tweenmax.to will take your element `to` left :0 bu, if you want to get it back it will only slide back if it's displaying. 


SO, 

One hacky solution is to just initialise the menu with `display:block` BUT better solution would be to use `add` class, `remove` class and NOT use toggle